coordinator/src/main/scala/com/socrata/datacoordinator/service/Main.scala

```diff
@@ -238,7 +238,17 @@ object Main {
           }
         }
 
-        val serv = new Service(serviceConfig, operations.processMutation, operations.processCreation, getSchema _,
+        def getRollups(datasetId: DatasetId) = {
```

Comment:
Yeah... that is a hard question to answer.  For now I'm just trying to follow the same patterns as exist for, say, getSchema.  For better or worse.